Automattic/wp-calypso
# Plugins Feature Gate

This PR adds a mocked example to the jetpack manage disabled warning in the plugins list. Also, creates a (dumb) component used to render a non clickable mocked example of any set of components, with some opacity gradient on top.

So, pretty much, transform this error screen:
![image](https://cloud.githubusercontent.com/assets/1554855/11422111/aaa277ae-9439-11e5-886b-dbc5ef2f78db.png)

Into this one:
![image](https://cloud.githubusercontent.com/assets/1554855/11422106/9afcd4d4-9439-11e5-8c87-152ed3e45431.png)


How to test:
=========
1. Go to any of your jetpack sites and disable Manage.
2. Go to http://calypso.localhost:3000/plugins
3. Switch to the manage disabled site
4. You should see the error screen
5. Make sure you can't interact anyway with the mocked components, neither with the mouse pointer or the keyboard.


And now, we can discuss the pros and the cons of this approach, @alternatekev @rickybanister @mtias @enejb @ebinnion
